purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -1011,27 +1073,6 @@ class PurchasesTest {
         }
     }
 
-    @Test
-    fun getOfferingsErrorIsNotCalledIfSkuDetailsMissing() {
```

Comment:
This test was incompatible with the new expected behavior, and with the behavior in iOS.
`configuration error if products are not set up` has the same setup, but checks for the existing error.

Let me know if I'm missing something?

Reply:
that makes sense to me..